### PR TITLE
feat: APP-177 add colors based on post file type in gallery/map view

### PR DESCRIPTION
**Pull request by blushi**
## Description

https://regennetwork.atlassian.net/browse/APP-177?atlOrigin=eyJpIjoiMjY1NDI1ZWVhZWFlNDg1M2FhOWE5ZmY5N2Q5MjQ2NWQiLCJwIjoiaiJ9

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

- Map: https://deploy-preview-2388--regen-storybook.netlify.app/?path=/story/organisms-postfiles--public
- Gallery: https://deploy-preview-2388--regen-storybook.netlify.app/?path=/story/organisms-postfiles--private-locations

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: tailwind.css [hunk 1]
```diff
@@ -78,6 +79,7 @@
     --blue-200: 216 228 245;
     --blue-100: 61 122 207;
     --blue-50: 61 122 207; /** replace with correct dark color ? **/
+    --blue-0: 61 122 207; /** replace with correct dark color ? **/
```
Comment (blushi) [inline tailwind.css:82]: @erikalogie dark color for 239 249 251? (light blue background color used for csv files)

**Comment by blushi** [comment]
@clevinson @erikalogie see testing instructions

**Review by aaronc — APPROVED** [review]
utACK

File: tailwind.css (hunk 1 above)
Comment (erikalogie) [reply tailwind.css:82]: Yes that is the correct dark color

**Comment by erikalogie** [comment]
The background for the green document type card should be #DCF0E3. Otherwise looks good!

**Comment by blushi** [comment]
> The background for the green document type card should be #DCF0E3. Also in the gallery view I think the icons should be colored for consistency. Otherwise looks good!

Just fixed this.

**Pull request merged**